src/utilcode/stgpool.cpp

```diff
@@ -361,7 +361,7 @@ bool StgPool::Grow(         // true if successful.
         return true;
     }
 
-#ifndef NO_CRT
+#if 0
```

Comment:
`#if 0` to match the existing behavior. It is unclear whether the code under this ifdef is good.